portfolio/src/main/java/com/google/sps/data/Login.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+import java.util.ArrayList;
+
+/** Class containing login information. */
+
+public final class Login {
+
+  // List of login info (email, login or logout link)
+  private final ArrayList<String> loginInfo = new ArrayList<>();
```

Comment:
Would this make more sense as structured data, rather than a list? Other than the comment it's not obvious what each element in the list represents.